src/components/Navigation/BottomNavigation.tsx

```diff
@@ -1,39 +1,72 @@
 import { BottomNavigationItem } from './NavigationItem';
-import { Link, useLocation } from 'react-router-dom';
+import { Link, useLocation, useNavigate } from 'react-router-dom';
+import { checkAuthUser } from '@/Api/user';
+import { useSetRecoilState } from 'recoil';
+import { informLoginModalState } from '@/store/store';
 
 const BottomNavigation = () => {
+  const setLogInModalOpened = useSetRecoilState(informLoginModalState);
   const { pathname } = useLocation();
+  const navigate = useNavigate();
+
+  const onHandlerLogInModal = (link: string, e: React.MouseEvent<HTMLAnchorElement>) => {
```

Comment:
```suggestion
  // click or open 
  const onClickLogInModal = (link: string, e: React.MouseEvent<HTMLAnchorElement>) => {
```
p5;

`onHandlerLogInModal`이름에서 handler가 명확하지 않은 네이밍 같아용. `click`처럼 일어나는 Event 이름으로 변경해도 좋겠어요.👍

Reply:
onOpenLogInModal로 변경했습니다!